client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -209,49 +245,113 @@ class CancelPurchase extends Component {
 					purchaseId={ this.props.purchaseId }
 				/>
 
-				<HeaderCake
-					backHref={ this.props.getManagePurchaseUrlFor(
-						this.props.siteSlug,
-						this.props.purchaseId
-					) }
+				<Button
+					compact
+					borderless
+					className="cancel-purchase__back-link"
+					href={ this.props.getManagePurchaseUrlFor( this.props.siteSlug, this.props.purchaseId ) }
 				>
-					{ titles.cancelPurchase }
-				</HeaderCake>
-
-				<Card className="cancel-purchase__card">
-					<h2>{ heading }</h2>
-
-					<CancelPurchaseRefundInformation
-						purchase={ purchase }
-						isJetpackPurchase={ isJetpackPurchase }
-						includedDomainPurchase={ this.props.includedDomainPurchase }
-						confirmBundledDomain={ this.state.confirmCancelBundledDomain }
-						cancelBundledDomain={ this.state.cancelBundledDomain }
-						onCancelConfirmationStateChange={ this.onCancelConfirmationStateChange }
-					/>
-				</Card>
-
-				<PurchaseSiteHeader siteId={ siteId } name={ siteName } domain={ siteDomain } />
-				<CompactCard className="cancel-purchase__product-information">
-					<div className="cancel-purchase__purchase-name">{ purchaseName }</div>
-					<div className="cancel-purchase__description">{ purchaseType( purchase ) }</div>
-					<ProductLink purchase={ purchase } selectedSite={ this.props.site } />
-				</CompactCard>
-				<CompactCard className="cancel-purchase__footer">
-					<div className="cancel-purchase__refund-amount">
-						{ this.renderFooterText( this.props ) }
+					<Gridicon icon="chevron-left" size={ 18 } />
+					{ this.props.translate( 'Back' ) }
+				</Button>
+
+				<FormattedHeader
+					className="cancel-purchase__formatter-header"
+					brandFont
+					headerText={ heading }
+					align="left"
+				/>
+
+				<div className="cancel-purchase__layout">
+					<div className="cancel-purchase__layout-col cancel-purchase__layout-col-left">
+						<PurchaseSiteHeader siteId={ siteId } name={ siteName } domain={ siteDomain } />
+
+						<CancelPurchaseRefundInformation
+							purchase={ purchase }
+							isJetpackPurchase={ isJetpackPurchase }
+							confirmBundledDomain={ this.state.confirmCancelBundledDomain }
```

Comment:
There's no `confirmBundledDomain` prop on `CancelPurchaseRefundInformation`. Did you mean `confirmCancelBundledDomain`?

Reply:
`confirmBundledDomain`  has been replaced with replaced with `confirmCancelBundledDomain`

Reply:
It hasn't, though. The code here still reads,

```
confirmBundledDomain={ this.state.confirmCancelBundledDomain }
```

But that prop, `confirmBundledDomain`, does not exist.